service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/submittedcase/SubmitCaseCafcassTemplate.java

```diff
@@ -1,11 +1,13 @@
 package uk.gov.hmcts.reform.fpl.model.notify.submittedcase;
 
-import lombok.Getter;
-import lombok.Setter;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.SuperBuilder;
 import uk.gov.hmcts.reform.fpl.model.notify.SharedNotifyTemplate;
 
-@Getter
-@Setter
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder
+@Data
 public final class SubmitCaseCafcassTemplate extends SharedNotifyTemplate {
-    private String cafcass;
+    String cafcass;
```

Comment:
is there a reason for this no longer being private?

Reply:
as discussed, using the `toBuilder` method to make the field private